actors/evm/src/interpreter/system.rs

```diff
@@ -61,15 +62,16 @@ impl<'r, RT: Runtime> System<'r, RT> {
             nonce: 1,
             saved_state_root: None,
             bytecode: None,
-            readonly: false,
+            readonly: read_only,
```

Comment:
Shouldn't be possible, but I like to be safe.

Reply:
doesnt hurt.